common_features.mk

```diff
@@ -237,6 +237,11 @@ ifeq ($(strip $(HD44780_ENABLE)), yes)
     OPT_DEFS += -DHD44780_ENABLE
 endif
 
+ifeq ($(strip $(VELOCIKEY_ENABLE)), yes)
```

Comment:
I'd much rather this be a simple define, but there isn't a good way to handle that.

That said, I think that this should be a sub set of the rgblight block, since it should only be enabled if rgblight is enabled. 

eg:
```c
ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
    OPT_DEFS += -DRGBLIGHT_ENABLE
    SRC += $(QUANTUM_DIR)/rgblight.c
    CIE1931_CURVE = yes
    LED_BREATHING_TABLE = yes
    ifeq ($(strip $(RGBLIGHT_CUSTOM_DRIVER)), yes)
        OPT_DEFS += -DRGBLIGHT_CUSTOM_DRIVER
    else
        SRC += ws2812.c
    endif
    ifeq ($(strip $(VELOCIKEY_ENABLE)), yes)
        OPT_DEFS += -DVELOCIKEY_ENABLE
        SRC += $(QUANTUM_DIR)/velocikey.c
    endif
endif
```

Otherwise, have you tested if velicikey is enabled and rgblight isn't?  
I suspect that the build will outright fail.

Reply:
Velocikey contains no references to rgblight - rgblight only optionally references velocikey if it is enabled, so the build is not impacted regardless of which are enabled.

This is a deliberate decision since some people have expressed interest in using Velocikey to control their key backlighting (which, if I understand correctly, is also independent of rgblight), so this keeps it reusable.